src/components/OpenLayersMap.js

```diff
@@ -23,6 +24,7 @@ class OpenLayersMap extends React.Component {
       source: new SourceVector({ wrapX: false }),
       map: null,
       mapInteractions: [],
+      rotation: 0,
```

Comment:
I don't think this needs to be in state. Just have `onRotationChange` set the value on the map.